easybuild/easyconfigs/g/GTK3/GTK3-3.24.37-GCCcore-12.3.0.eb

```
@@ -0,0 +1,78 @@
+easyblock = 'Bundle'
+
+name = 'GTK3'
+version = '3.24.37'
+
+homepage = 'https://developer.gnome.org/gtk3/stable/'
+description = """GTK+ is the primary library used to construct user interfaces in GNOME. It
+ provides all the user interface controls, or widgets, used in a common
+ graphical application. Its object-oriented API allows you to construct
+ user interfaces without dealing with the low-level details of drawing and
+ device interaction.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+builddependencies = [
+    ('binutils', '2.40'),
```

Without Autotools installed in the OS the build failes with: `/bin/bash: line 1: autoreconf: command not found`. I would recommend to add Autotools to the build dependencies.

```suggestion
    ('binutils', '2.40'),
    ('Autotools', '20220317'),
```